asreview/project.py

```diff
@@ -59,60 +63,126 @@ class ProjectNotFoundError(Exception):
     pass
 
 
-def get_project_path(project_id, asreview_dir=None):
+# !=============================================
+# @TODO: I HATE THE FACT THAT I AM TALKING ABOUT
+# Users in this module, but I need a quick fix
+# for now
+# !=============================================
```

Comment:
I don't think there is a way around talking about users here. I also don't really mind it.